### ISSUE TITLE
lolcat doesn't seem to play nice with '\b' (backspace) or "\33[2K\r" (clear console line)

### ISSUE THREAD

**Issue by n-ham** [busyloop/lolcat#110]
The '\b' character is often used for backspace and "\33[2K\r" is used to clear the console line on unix machines. Neither of which lolcat seems to play nicely with. It would be awesome if you were able to update lolcat so it does..

**Comment by tbooth** [comment]
Yeah I was hoping to make all my manpages look nice with something like:

`$export MANPAGER='bash -c "lolcat -f \"\$@\" | less -R" PAGER'`

But I just start seeing all the ^H control codes. Meh.

**Comment by n-ham** [comment]
I might have you sorted.. I ended up making my own version of lolcat which is ~4 times faster than the c (old world's fastest). I haven't release it yet (will in a few days), but it's in-built in to Nift (https://nift.dev). Eg. install Nift then `nift lolcat-cc` for lolcat-cc command (can even do `nift sh` then `lolcat.on` and you will get rainbow colours for output by default :).

**Comment by m-o-e** [comment]
@n-ham @tbooth 
lolcat tries to deal with control sequences to a degree
but it's tricky and it doesn't always succeed.

This may be improved in the future (PRs always welcome 😬).

In most cases you can help it a little by stripping the
problematic sequences before feeding the input to lolcat.

For man-pages you can use this:

`export MANPAGER='bash -c "col -bx | lolcat -f \"\$@\" | less -R" PAGER'`

**Comment by alexmyczko** [comment]
HERE BE DRAGONS. another trick is bash | lolcat (NO WARRANTIES)
that works ok good with mc, but not so much with modplug123, nor ocp
